app/src/main/java/com/example/parkingsystem/fragment/ConfigureParkingDialogListener.java

```diff
@@ -0,0 +1,7 @@
+package com.example.parkingsystem.fragment;
```

Comment:
this should be into a listener package at the same level of the fragment package, please move it

Reply:
Moved!